assignments/lambda-classes.js

```diff
@@ -1 +1,210 @@
 // CODE here for your Lambda Classes
+/*
+Lambda personnel can be broken down into three different types of people.
+
+    Instructors - extensions of Person
+    Students - extensions of Person
+    Project Managers - extensions of Instructors
+
+
+*/
+
+class Person {
+    constructor(args) {
+        const { name, age, location, gender } = args;
```

Comment:
Yes, perfect usage of destructuring, loving it, great job Pavol !!! 🔥 

You can also do it even shorter way:

```
class Person {
    constructor({ name, age, location, gender }) {
        this.name = name;
        this.age = age;
        this.location = location;
        this.gender = gender;
    }
```